chromadb/proto/convert.py

```diff
@@ -309,8 +333,341 @@ def from_proto_request_version_context(
 
 def to_proto_request_version_context(
     request_version_context: RequestVersionContext,
-) -> proto.RequestVersionContext:
-    return proto.RequestVersionContext(
+) -> chroma_pb.RequestVersionContext:
+    return chroma_pb.RequestVersionContext(
         collection_version=request_version_context["collection_version"],
         log_position=request_version_context["log_position"],
     )
+
+
+def to_proto_where(where: Where) -> chroma_pb.Where:
```

Comment:
Is this copy pasted from somewhere?

Reply:
This is copy pasted from the old grpc metadata segment impl, which will be removed in the future